CMakeLists.txt

```diff
@@ -511,7 +511,11 @@ ENDIF()
        IF("${OPENSSL_ROOT_DIR}" STREQUAL "")
           get_filename_component(OPENSSL_ROOT_DIR "${OPENSSL_INCLUDE_DIR}/.." REALPATH)
        ENDIF()
-       SET(OPENSSL_CONF_SOURCE "${OPENSSL_ROOT_DIR}/ssl/openssl.cnf")
+	   IF("${OPENSSL_ROOT_DIR}" STREQUAL "C:/Program Files/OpenSSL")
+		  SET(OPENSSL_CONF_SOURCE "C:/Program Files/Common Files/SSL/openssl.cnf")
```

Comment:
Thank you anyway.
* Hard-coded driver symbol and path are not good practice, because not everyone installs the files to the same path even though it's default behavior of installer. Better approach would be to add a new variable for location of `openssl.cnf` file, E.G. `OPENSSL_CNF_FILE_DIR`. Perhaps best approach is to detect whether the file exists in one location, if not, try another location.
* A minor issue: need to replace the tabs in the code with white spaces.